__tests__/app.test.js

```diff
@@ -95,3 +96,35 @@ describe("/api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/articles", () => {
+  test("200: GET /api/articles should return an array of article objects with correct keys", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.articles.length).toBe(5);
+        body.articles.forEach((article) => {
+          expect(article).toMatchObject({
+            author: expect.any(String),
+            title: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+            comment_count: expect.any(String),
+          });
```

Comment:
This is a really good, but because you are using an exact number when checking the length in the first expect statement this test is not as flexible as it could be, instead I would recommend looking into the jest matcher .toBeGreaterThan()